src/amount.h

```diff
@@ -22,7 +22,7 @@ static const CAmount COIN = 100000000;
  * critical; in unusual circumstances like a(nother) overflow bug that allowed
  * for the creation of coins out of thin air modification could lead to a fork.
  * */
-static const CAmount MAX_MONEY = 21000000 * COIN;
+static const CAmount MAX_MONEY = 10000000000 * COIN; // Dogecoin: maximum of 100B coins (given some randomness), max transaction 10,000,000,000
```

Comment:
```suggestion
static const CAmount MAX_MONEY = 10000000000 * COIN; // Dogecoin: maximum of 10B coins (given some randomness), max transaction 10,000,000,000
```

Reply:
```suggestion
static const CAmount MAX_MONEY = 10000000000 * COIN; // Dogecoin: total supply is not finite, but we constrain maximum single transaction to 10,000,000,000
```
Ah, it's meant to refer to total supply, and I realise the comment is a legacy from Dogecoin 0.4.